CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## X.Y.Z 2022-xx-xx
 ### PaymentSheet
 * [Fixed] Fixed user facing error messages for card related errors.
+* [Fixed] Fixed `setup_future_usage` value being overridden for new customers when it was already set by the Payment Intent.
```

Comment:
I wouldn't say 'new customers' - this is fixing the case where there is no customer.

Maybe like
* [Fixed] Fixed `setup_future_usage` value being set when there's no customer.